draft-ietf-quic-applicability.md

```diff
@@ -451,16 +451,21 @@ firewalls that rely on the port number for application identification.
 
 # Connection Migration
 
-QUIC supports connection migration. Even if lower-layer addresses (usually the
-4-tuple of IP addresses and ports) changes, QUIC packets can still be associated
-with an existing connection based on the Connection ID (see also section
-{{connid}}) in the QUIC header, if present. This supports cases where address
-information changes due to e.g. NAT rebinding or change of the local interface.
+QUIC supports connection migration by the client. If a lower-layer address
+changes, a QUIC endpoint can still associate packets with an existing connection
+based on the Connection ID (see also section {{connid}}) in the QUIC header,
+if present. This supports cases where address information changes due to e.g.
+NAT rebinding, intentional change of the local interface, or based on an
+indication in the handshake of the server for a preferred address to be used.
+
 Currently QUIC only supports failover cases. Only one "path" can be used at a
 time, and as soon as the new path is validated all traffic will be switched over
-to the next path. Of course if an endpoint decided to not use the Connection ID
-in short packets (Zero-length Conn ID) for a certain connection, migration is
-not supported for that direction of the connection.
+to the next path. For intentional migrations the client can send probing packets
+before switching over to measure the path characteristics and eventually pre-heat
```

Comment:
"pre-heat" is a colloquialism that should be avoided.  Note that probing does very little to allow the congestion window to open, so this is questionable anyway.

Reply:
We discussed this ones in a multipath related discussion. The idea here was to send multiple probing packets (and II guess padding). If that is seen as a viable approach for e.g. mobile handovers, it would be good to mention is somewhere. However, I guess we would then need to say more. I believe @erickinnear brought this up... otherwise if people think it's too early to say anything like that,  can also remove the half sentence.

Reply:
I'm in favour of not saying anything after the /and/.

Reply:
@gorryfair this PR was merged a long time ago. let's have the discussion in issue #197. As you can see that issue is flagged for meeting discussion.